anuonifade/loanapp
# #150952229 & #150952223 - New user add profile and existing user update profile

#### What does this PR do?
This PR redirects new user to update profile to add their profile and existing user can edit and update different sections of their profile.

#### Description of task to be completed?
- A logged in user for the first time would be redirected to profile update page
- When an existing user click on edit profile, they should be able to see the page to update different section of their profile.

#### How should this be manually tested?
- Register as a new user
- Login to the app and you should be redirected to the profile update page OR
- Log in as an existing user
- Go to profile page and click on edit profile
- Update any section of the profile page and the profile should be updated.

#### What are the relevant pivotal tracker stories?
#150952229 & #150952223

#### Screenshot
![screen shot 2017-12-13 at 7 43 06 pm](https://user-images.githubusercontent.com/26273055/33956157-030c9f9e-e03e-11e7-83c7-a8651355fcc5.png)
<img width="962" alt="screen shot 2017-12-13 at 7 43 17 pm" src="https://user-images.githubusercontent.com/26273055/33956165-0820d6da-e03e-11e7-8dfa-965d6df40eee.png">
